apps/vs-code-designer/src/app/utils/vsCodeConfig/settings.ts

```diff
@@ -21,7 +21,7 @@ import type { WorkspaceConfiguration, WorkspaceFolder } from 'vscode';
 export function getGlobalSetting<T>(key: string, prefix: string = ext.prefix): T | undefined {
   const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration(prefix);
   const result: { globalValue?: T; defaultValue?: T } | undefined = projectConfiguration.inspect<T>(key);
-  return result && (result.globalValue || result.defaultValue);
+  return result && (result.globalValue ?? result.defaultValue);
```

Comment:
Is there a reason why we don't just do `return result?.globalValue ?? result?.defaultValue;`?

Reply:
Not really. I think its just two different ways to write it down. Do you feel that its better this way? `return result?.globalValue ?? result?.defaultValue;`

Reply:
ah I see, I think it's okay then!